test/new_tests/test_free_threading.py

```diff
@@ -0,0 +1,40 @@
+import threading
+import pytest
+import aerospike
+
+
+@pytest.mark.usefixtures("as_connection")
+class TestFreeThreading:
+    def test_unsafe(self):
+        key = ("test", "demo", 1)
```

Comment:
Probably not a great idea to only use one key - you should generate a bunch of keys - the number of keys / number of operations will give you the basic collision rate.